.github/workflows/release.yml

```diff
@@ -29,6 +29,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@master
```

Comment:
As discussed separately, our understanding is that for tag pushes this action will check out the tag. Please add a comment here to remind us of this fact.